### ISSUE TITLE
About traing loss trend

### ISSUE THREAD

**Issue by zsy0828** [google-deepmind/graphcast#103]
Hi, I train the graphcast model, I'd like to know the mseloss drop trend during training the model.
I train only in 925, 850, 700, 500, 200, 100hpa, and surface, but I didn't use the variables which are same as paper. I used the hid_dim=512 and num_mesh_gnn_layer=16, but i'm afraid that i write the wrong code. Thus, if possible, could you illustrate the loss trend during training the graphcast_large on the one step pred?
the optimizer and loss function is same as the paper and your code.
I use the 2 * A800, the data is from 2002 to 2022, in the first 1k steps, the loss is train from 4~9 to 1.5~1.7, is this trend is closed to your training? 
Because I use hid_dim=32 and num_mesh_gnn_layer=4, training the same data with same optimizer and loss, In the first 1k steps, the loss is train from 4~9 to 0.6~0.8, but in the seconde 2steps, the loss is luctuated to 1.0~1.1.
So, If possible, could you share the loss drop trend, it will help me to judge my code is wrong or not.
